src/flowfuse-dashboard.njk

```diff
@@ -0,0 +1,119 @@
+---
```

Comment:
Why not `/src/product/dashboard.njk`?

Reply:
I think that's @Hasmin-AC's point above, I agree that it makes sense to put it there.

Reply:
Done!